integration-tests/common.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```

Comment:
@fridex it works, TYVM. Should I just open new PR with changed link in README.md and git rm of common.html?